src/Dashboard/Newsfeed/Store.js

```diff
@@ -1,26 +1,26 @@
-import _ from 'lodash/fp'
 import MCInfo from './MCInfo'
 import PtmInfo from './PtmInfo'
 import StationsInfo from './StationsInfo'
-import { selectors as ExamsSelectors, actions as ExamsActions } from '../../Exams/Store'
-import { selectors as StationsSelectors } from '../../Exams/Stations/Store'
-import { selectors as MCSelectors } from '../../Exams/MC/Store'
-import { selectors as PtmSelectors } from '../../Exams/Ptm/Store'
-import { color as stationsColor } from '../../Exams/Stations/Stations'
-import { color as mcColor } from '../../Exams/MC/MC'
-import { color as ptmColor } from '../../Exams/Ptm/Ptm'
+import {selectors as ExamsSelectors, actions as ExamsActions} from '../../Exams/Store'
+import {selectors as StationsSelectors} from '../../Exams/Stations/Store'
+import {selectors as MCSelectors} from '../../Exams/MC/Store'
+import {selectors as PtmSelectors} from '../../Exams/Ptm/Store'
+import {color as stationsColor} from '../../Exams/Stations/Stations'
+import {color as mcColor} from '../../Exams/MC/MC'
+import {color as ptmColor} from '../../Exams/Ptm/Ptm'
+import {flow, over, map, merge, groupBy} from '../../Utils/utils'
 
-const getData = _.flow([
-    _.over([ 
-        _.flow([ StationsSelectors.getTimeline, _.map(_.merge({ icon: 'PP', color: stationsColor, comp: StationsInfo }))]),
-        _.flow([ MCSelectors.getTimeline, _.map(_.merge({ icon: 'MC', color: mcColor, comp: MCInfo }))]),
-        _.flow([ PtmSelectors.getTimeline, _.map(_.merge({ icon: 'PTM', color: ptmColor, comp: PtmInfo }))]),
+const getData = flow([
+    over([
+        flow([StationsSelectors.getTimeline, map(merge({icon: 'PP', color: stationsColor, comp: StationsInfo}))]),
+        flow([MCSelectors.getTimeline, map(merge({icon: 'MC', color: mcColor, comp: MCInfo}))]),
+        flow([PtmSelectors.getTimeline, map(merge({icon: 'PTM', color: ptmColor, comp: PtmInfo}))]),
```

Comment:
Ehrlich gesagt, verstehe ich kein wort von diesen Zeilen. Vor allem map(merge({...})) macht überhaupt keinen Sinn. 

Also diese Stelle würde ich nicht auf diese Art refactoren. Hier wird zwar lodash herausgenommen, aber rein syntaktisch, ohne dass sich die Lesbarkeit des Codes verbessert. 

In der Refactoring-Anleitung steht ja bei lodash "Der Code sollte keine Lodash-Abhängigkeiten haben. Zwischenergebnisse sollen in Variablen mit verständlichen Namen abgebildet sein." 
Bitte vorfolge mal nach, was für einen Sinn dieses flow, map und merge hat, und mit diesem Wissen führe bitte sinnvoll bekannte Variablen für zwischenergebnisse ein.
D.h. getData wird zu einer mehrzeiligen function werden.